DS/MainArrayList.java

```diff
@@ -0,0 +1,274 @@
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+//this class is used to define an array list and its functionality
+class ArrayList {
+	private int[] arrayList,arrayListBuffer;
+	private int size=10,index;
+	//constructor of class to initialize an empty list
+	public ArrayList() {
+		index=0;
+		arrayList=new int[10];
+	}
+	//this method is used to add an item in list
+	public void addElementInList() {
+		Scanner sc=new Scanner(System.in);
+		System.out.println("Enter element");
+		int arrayElement=sc.nextInt();
+		if(index==size) {
```

Comment:
Size will always remain same (i. e. 10). So this condition will only true once when 'index' will hit higher limits. It will not work after that.